samples/githubDispatch/package.json

```diff
@@ -0,0 +1,9 @@
+{
+    "engines": {
+      "node": ">= 14"
+    },
+    "dependencies": {
+      "superagent": "6.1.0",
+      "octokit": "1.0.5"
+    }
+  }
```

Comment:
Can you also update the main README.md with a link to this sample at https://github.com/fusebit/fusebot#samples?

Reply:
Done